CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 
 - ds-rshiny upgrade and housekeeping ([#563](https://github.com/opendevstack/ods-quickstarters/issues/563))
 - ds-jupyter-notebook renamed to ds-jupyter-lab, upgrade to JupyterLab 3 and UBI8 base image introduction ([#562](https://github.com/opendevstack/ods-quickstarters/issues/562))
```

Comment:
Is it intended to have the jupyter related changes description here as there are no related changes I can see.

Reply:
that is an update of the previous changelog changes related to that jupyter PR (it is not added but moved), it is a changelog refactor :P